ui/tests/integration/components/form-field-test.js

```diff
@@ -343,6 +349,333 @@ module('Integration | Component | form field', function (hooks) {
     await render(
       hbs`<FormField @attr={{this.attr}} @model={{this.model}} @modelValidations={{this.validations}} @onChange={{this.onChange}} />`
     );
-    assert.dom('[data-test-validation-warning]').exists('Validation warning renders');
+    assert.dom('[data-test-form-field-validation-warning]').exists('Validation warning renders');
+  });
+
+  // ---------------
+  // HDS FORM FIELDS
+  // ---------------
+  // Note: some tests may be duplicative of the generic tests above
+  //
+
+  // ––––– editType === 'checkboxList' / possibleValues –––––
+
+  test('it renders: editType=checkboxList / possibleValues - as Hds::Form::Checkbox::Group', async function (assert) {
+    const possibleValues = ['foo', 'bar', 'baz'];
+    await setup.call(this, createAttr('foo', '-', { editType: 'checkboxList', possibleValues }));
+    assert
+      .dom('.field [class^="hds-form-group"] input[type="checkbox"].hds-form-checkbox')
+      .exists('renders as Hds::Form::Checkbox::Group');
+    assert
+      .dom('fieldset[data-test-input="foo"] input[type="checkbox"]')
```

Comment:
Hmm... I like the approach here in that it simplifies a lot, but can we use the GENERAL selectors with the `fieldset`?
```suggestion
      .dom('fieldset[GENERAL.inputByAttr("foo")] input[type="checkbox"]')
```

Reply:
TODO